tsreport.c

```diff
@@ -476,6 +477,10 @@ static int report_buffering_stats(TS_reader_p  tsreader,
       const int is_discontinuity = (adapt != NULL && (adapt[0] & 0x80) != 0);
       struct stream_data * const ss = stats + index;
 
+      if (printcc)
+        fprint_msg("CC %d\n", cc);
```

Comment:
There is a lot more that could be logged, using the -v switch will add a lot. However I just did this purely as a test of changes to Bento to make sure that the CC's are as expected so it can be used in a regression test